recipe/meta.yaml

```diff
@@ -20,15 +20,15 @@ requirements:
     - nlopt
     - boost     1.63.*
     - numpy     x.x
-    - dill
+    - cloudpickle
```

Comment:
Interesting: why did you decide to move from dill to cloudpickle? This is just out of curiosity because in Spyder we're thinking to use one of them in the future.